index.html

```diff
@@ -0,0 +1,101 @@
+        <path d="M403.692 150.602L288.021 224.919" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M438.197 250.626L306.642 276.332" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M439.95 357.594L308.286 331.766" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M408.513 458.836L292.841 384.519" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M347.939 542.047L262.061 428.135" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M265.456 597.237L219.779 457.496" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M166.215 564.707L166.215 468.947" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M75.8476 600.89L121.634 461.149" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-8.27754 548.869L77.5997 435.078" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-71.4806 468.094L44.3004 393.777" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-105.984 368.193L25.6795 342.365" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-107.628 261.104L24.0361 286.932" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-76.3007 159.862L39.4802 234.179" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M-15.7264 76.6514L70.2604 190.564" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        <path d="M66.8653 21.583L112.542 161.202" stroke="#A5ADBA" stroke-width="0.426199" stroke-miterlimit="10" stroke-linecap="round" stroke-dasharray="0.43 0.43"/>
+        </svg>
+        
+    <div class="decor">
+        <img src="images/polygon1.png" alt="">
+        <img src="images/box1.png" alt="">
+        <img src="images/polygon2.png" alt="">
+        <img src="images/box2.png" alt="">
+        <img src="images/cylinder2.png" alt="">
+        <img src="images/cylinder1.png" alt="">
+    </div>
```

Comment:
Please notice that these images (inside "decor") shouldn't be an `img` element. Instead, it should be the [background image](https://www.w3schools.com/cssref/pr_background-image.php) of your container. This will make your code semantically accurate and give you an easier time with styling 😃. You can [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information.